account_statement_import_txt_xlsx/models/account_statement_import_sheet_mapping.py

```diff
@@ -123,6 +172,18 @@ class AccountStatementImportSheetMapping(models.Model):
     bank_account_column = fields.Char(
         help="Partner's bank account",
     )
+    footer_lines_count = fields.Integer(
+        string="Footer lines number",
+        help="Set the Footer lines number."
+        "Used in some csv file that integrate meta data in"
```

Comment:
This can be used for other things apart from CSV, so check if it's being used everywhere. If not, include it, and change the label.

Reply:
And the name is misleading. It should `footer_lines_skip_count`.